### DISCUSSION TITLE
When will the SSL certificate of GitHub.com be updated?

### DISCUSSION THREAD

**Discussion by anfisc** [community/community#47540] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The public SSL certificate of github.com is only valid until March 16th 2023 and 00:59:59.   
Question:   
When is it planned to provide a certificate with a longer validity?   
Since the current certificate is valid as of March 15th 2022 I assume that the replacement of the certificate is planned for the near future.   
Kind regards,  
Andre

**Comment by unknowVariable** [comment]
Thank you for your product feedback regarding the validity of the SSL certificate for github.com. It's great to see that you're keeping an eye on the validity of the certificate and its expiration date.

GitHub, like many other companies, typically updates their SSL certificates well before they expire, to ensure that there is no disruption in service. This means that it's likely that GitHub already has plans to provide a certificate with a longer validity, but the exact timing of this update may not be publicly available.

If you have any concerns about the expiration of the SSL certificate and its potential impact on your use of GitHub, I would recommend reaching out to GitHub's support team directly. They may be able to provide more information about their plans for updating the certificate and any potential impact on users.
